src/main/java/jenkins/plugins/http_request/HttpRequest.java

```diff
@@ -281,19 +320,28 @@ private HttpRequestBase getHttpRequestBase(PrintStream logger, RequestAction req
         return httpRequestBase;
     }
 
-    private FilePath getOutputFilePath(AbstractBuild<?, ?> build) {
+    private FilePath getOutputFilePath(FilePath workspace, PrintStream logger) {
+        // No need to check for a null workspace, the workflow node step prevents workspace from being null.
```

Comment:
Anyway, `getOutputFilePath` seems to not be called from `HttpRequestStep$Execution.run` so I guess it's ok like it is now.